libs/designer/src/lib/core/utils/parameters/__test__/helper.spec.ts

```diff
@@ -510,7 +510,7 @@ describe('core/utils/parameters/helper', () => {
           {
             id: '6',
             type: ValueSegmentType.TOKEN,
-            value: 'item()?[\'someItem\']',
```

Comment:
Does these changes mean that something happened/changed with the original function that the test ended up updating it?

Reply:
I tried changing this back to what it was here, and running prettier on it. The result is the same (back to double quote). I changed it back again and ran 'lint' but it did not change the file (kept single quote). I think the previous commit must have been made using some difference in prettier, either config or version. Worth looking into but I think this can still be merged. Came across this when researching... https://github.com/prettier/prettier/issues/973